pkg/pool-hooks/test/foundry/HyperSurgeLiquidityChecks.t.sol

```diff
@@ -999,7 +999,7 @@ contract HyperSurgeLiquidityCheckTest is BaseVaultTest, HyperSurgeHookDeployer,
         );
         vm.stopPrank();
 
-        assertTrue(ok, "must be greater than, equal is fine");
+        assertFalse(ok, "must be greater than, equal is fine");
```

Comment:
hmm why did this change?

Reply:
This case starts 4.9% below the price, and ends 5.2% above the price. The threshold of the deviation is 0.1%, so it's above the threshold.
The price deviation is worsening, which should be blocked by the hook, given that it's an unbalanced liquidity operation.